internal/cache/file/downloader/parallel_downloads_job.go

```diff
@@ -32,15 +33,15 @@ import (
 // GCS into given destination writer.
 //
 // This function doesn't take locks and can be executed parallely.
-func (job *Job) downloadRange(ctx context.Context, dstWriter io.Writer, start, end int64) error {
+func (job *Job) downloadRange(ctx context.Context, dstWriter io.Writer, start, end uint64) error {
```

Comment:
Use unsigned ints sparingly: https://yaqs.corp.google.com/eng/q/8035320036228857856

Reply:
minObject.Size is uint64. Using uint64 to keep it consistent with that.

Reply:
It seems like a step in the reverse direction - the size in the MinObj is uint64 which is assigned from ObjectAttrs.Size, an int64 here.
For this PR, I think we should use continue using int64 and add a TODO to change MinObj.Size to int64 later.

Reply:
Thanks . Missed it. updated now.